src/Layers/xrRender/R_Backend.h

```diff
@@ -358,6 +358,7 @@ class ECORE_API CBackend
     void set_ClipPlanes(u32 _enable, Fplane* _planes = nullptr, u32 count = 0);
     void set_ClipPlanes(u32 _enable, Fmatrix* _xform = nullptr, u32 fmask = 0xff);
     IC void set_Scissor(Irect* rect = nullptr);
+    IC void SetViewport(D3D_VIEWPORT viewport) const;
```

Comment:
I think it's not very good to pass structure, not a reference of structure

Reply:
I think it's even better to pass r-value reference.

Reply:
Yeah. And probably it’s time to rename ‘D3D_VIEWPORT’ into smth like ‘Viewport’

Reply:
I think it's a good name, which shows that DirectX is a head renderer and other backends implementations should mimic to DirectX behaviour.

Reply:
And personally I can't even imagine a good name for the type that would fit.

`Viewport` could be good, but...
```cpp
// public member variables should be PascalCase

struct ThatWasMadeForSomeReason
{
	SomeType SomeVar;
	... // more variables
	Viewport Viewport; // I'm not sure, but GCC won't be happy about this probably
                       // But even if it will, this looks bad.
}